src/components/header/index.js

```diff
@@ -38,7 +39,7 @@ export const Header = () => {
 
   return (
     <Box>
-      <Flex
+      <Stack
```

Comment:
@cisaacstern, i'm curious about this change from `Flex` to `Stack`. do we still need it?

Reply:
@andersy005, when I was prototyping this, using `Stack` was the only way I could find to get the navigation header and the alert to stack vertically. With `Flex`, I believe they were aligned horizontally. If we switch `Stack` back to `Flex`, will the alert banner still be aligned vertically, as it is now?

Reply:
> If we switch Stack back to Flex, will the alert banner still be aligned vertically, as it is now?

yes. the banner is currently outside of the `<Stack></Stack>` tag

Reply:
In that case, no reason to keep `Stack` 😄

Reply:
Reverted back to `Stack`